easybuild/easyblocks/p/psi.py

```diff
@@ -121,7 +87,59 @@ def configure_step(self):
         env.setvar('PSI_OBJ_INSTALL_DIR', self.install_psi_objdir)
         env.setvar('PSI_SRC_INSTALL_DIR', self.install_psi_srcdir)
 
-        super(EB_PSI, self).configure_step(cmd_prefix=self.cfg['start_dir'])
+        # explicitely specify Python binary to use
+        pythonroot = get_software_root('Python')
+        if not pythonroot:
+            self.log.error("Python module not loaded.")
+        env.setvar('PYTHON', os.path.join(pythonroot, 'bin', 'python'))
+
+        # They have a non-standard variable to define the install path
+        env.setvar('PREFIX', self.installdir)
```

Comment:
Older version also had an CMake based build system, but it was optional.

I don't see the harm in doing this in all cases?